ingestion/functions/parsing/brazil_paraiba/sample_data.csv

```diff
@@ -0,0 +1,2 @@
+M�todo,Munic�pio de Resid�ncia,Inicio Sintomas,Sexo,Doen�as preexistentes,Idade,Data do �bito
```

Comment:
is github messing with the encoding? These should appear fine if they are utf-8 encoded.

Reply:
I really don't know what it going on with the encoding, it seems like either there is an encoding error or there is a KeyError where the key in paraiba.py is not being matched to the column in sample_data.csv

Reply:
In vscode you can check the encoding at the bottom right corner of the window (here UTF-8 on my screenshot below), if you click here you can save the file as the desired encoding, please try that

![image](https://user-images.githubusercontent.com/1255432/94013059-d4389600-fda9-11ea-9946-805c6364a4a4.png)